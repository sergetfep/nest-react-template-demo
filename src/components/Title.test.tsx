import React from "react";
import renderer from "react-test-renderer";
import { Title } from "./Title";

describe("Title Component", () => {
  it("should match snapshot", () => {
    const component = renderer.create(<Title />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
