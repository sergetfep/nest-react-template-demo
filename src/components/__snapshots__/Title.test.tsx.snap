// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title Component should match snapshot 1`] = `
<h1
  className="jsx-516677764 title"
>
  Welcome to 
  <a
    className="jsx-516677764"
    href="https://nextjs.org"
  >
    Next.js!
  </a>
</h1>
`;
